packages/dashboard/src/components/ConfirmDialog/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { MaskDialog } from '@dimensiondev/maskbook-theme'
+import {
+    DialogContent,
+    DialogActions,
+    Button,
+    dialogActionsClasses,
+    dialogContentClasses,
+    experimentalStyled as styled,
+    buttonClasses,
+} from '@material-ui/core'
+
+const StyledDialogActions = styled(DialogActions)(({ theme }) => ({
+    [`&.${dialogActionsClasses.root}>:not(:first-of-type)`]: {
+        marginLeft: theme.spacing(3),
+    },
+}))
+
+const StyledDialogContent = styled(DialogContent)(() => ({
```

Comment:
Since this component is only used once, I'd prefer to use the `sx` prop to do the work.

Reply:
I don't think the once used component should always use `sx` prop. We should keep the template clear. If the css properties for the component are more than 3, the styled component would be better.

Reply:
reasonable, but I don't like `[`&.${dialogContentClasses.root}`]`, it's looking tricky. It's ok to keep this one.